src/main/java/com/alkemy/ong/mapper/MemberMapper.java

```diff
@@ -5,25 +5,27 @@
 
 public class MemberMapper {
 
-    public static Member mapMemberModelToMember(MemberModel memberModel){
-        Member newMember = new Member();
-        newMember.setName(memberModel.getName());
-        newMember.setFacebookUrl(memberModel.getFacebookUrl());
-        newMember.setInstagramUrl(memberModel.getInstagramUrl());
-        newMember.setLinkedinUrl(memberModel.getLinkedinUrl());
-        newMember.setImage(memberModel.getImage());
-        newMember.setDescription(memberModel.getDescription());
-        return newMember;
+    public static Member mapModelToDomain(MemberModel memberModel){
+        Member memberDomain = Member.builder().
```

Comment:
Pasemos los puntos al comienzo de cada renglón, ejemplo:
```suggestion
        Member memberDomain = Member.builder()
        .id(...)
        .name(...)
```

Reply:
Lo mismo con el otro método.